flood_adapt/object_model/site.py

```diff
@@ -1,88 +1,153 @@
-import flood_adapt.object_model.validate.config as val
-from flood_adapt.object_model.io.config_io import read_config
+from enum import Enum
+from pathlib import Path
+from typing import Optional, Union
+
+import tomli
+import tomli_w
+from pydantic import BaseModel
+
+from flood_adapt.object_model.io.unitfulvalue import (
+    UnitfulDischarge,
+    UnitfulLength,
+    UnitTypesLength,
+)
+
+
+class Cstype(str, Enum):
+    """class describing the accepted input for the variable cstype in SiteConfig"""
+
+    projected = "projected"
+    spherical = "spherical"
+
+
+class Floodmap_type(str, Enum):
+    """class describing the accepted input for the variable floodmap in SiteConfig"""
+
+    water_level = "water_level"
+    water_depth = "water_depth"
+
+
+class SfincsModel(BaseModel):
+    """class describing the accepted input for the variable sfincs in SiteConfig"""
+
+    csname: str
+    cstype: Cstype
+    version: str
+    offshore_model: str
+    overland_model: str
+    datum_offshore_model: str
+    datum_overland_model: str
+    diff_datum_offshore_overland: UnitfulLength
+    tidal_components: str
+    ambient_air_pressure: float
+    floodmap_no_data_value: float
+    floodmap_units: UnitTypesLength
+
+
+class SlrModel(BaseModel):
+    """class describing the accepted input for the variable slr in SiteConfig"""
+
+    vertical_offset: UnitfulLength
+    relative_to_year: int
+
+
+class GuiModel(BaseModel):
+    """class describing the accepted input for the variable gui in SiteConfig"""
+
+    tide_harmonic_amplitude: UnitfulLength
+
+
+class RiskModel(BaseModel):
+    """class describing the accepted input for the variable risk in SiteConfig"""
+
+    flooding_threshold: UnitfulLength
+    return_periods: list
+
+
+class DemModel(BaseModel):
+    """class describing the accepted input for the variable dem in SiteConfig"""
+
+    filename: str
+    units: UnitTypesLength
+    indexfilename: str
+
+
+class FiatModel(BaseModel):
+    """class describing the accepted input for the variable fiat in SiteConfig"""
+
+    exposure_crs: str
+    aggregation_shapefiles: str
+    aggregation_field_names: str
+    floodmap_type: Floodmap_type
+
+
+class RiverModel(BaseModel):
+    """class describing the accepted input for the variable river in SiteConfig"""
+
+    name: str
+    long_name: str
+    mean_discharge: UnitfulDischarge
+    x_coordinate: float
+    y_coordinate: float
```

Comment:
we may need to handle multiple rivers, just add a to do here and we will tackle it later

Reply:
you add a to do like this: #TODO bla bla